abba/json/src/Parse.cpp

```diff
@@ -0,0 +1,480 @@
+#include "Parse.hpp"
+
+Parse::Parse(std::string filename) : json(filename)
+{
+
+}
+
+void Parse::parse()
+{
+	_file = &json.getFile();
+	std::cout << "i'm here" << std::endl;
+	getToken();
+	Json *root = new Json();
+	root->type = Json::OBJECT;
+	root = parseObject(root);
+	std::cout << "DONE PARSING!!!!!!!!!!!!" << std::endl;
+}
+
+char Parse::getWithoutWhiteSpace()
```

Comment:
This whole function can essentially be replaced with `_file >> std::ws`, see: https://en.cppreference.com/w/cpp/io/manip/ws